lib/v2/xmut/jwc/bkjw.js

```diff
@@ -0,0 +1,59 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const xmut = 'https://jwc.xmut.edu.cn';
+
+module.exports = async (ctx) => {
+    const { category = 'jwxt' } = ctx.params;
+    const url = `${xmut}/index/tzgg/${category}.htm`;
+    const res = await got(url, {
+        headers: {
+            referer: xmut,
+        },
+        https: {
+            rejectUnauthorized: false,
+        },
+    });
+    const $ = cheerio.load(res.data);
+    const itemsArray = $('#result_list table tbody tr').map((index, row) => {
+        const res = $('td', row).eq(0);
+        const resDate = $('td', row).eq(1);
+        const resLink = $('a', res).attr('href');
+        let link;
+        if (resLink.startsWith('../../')) {
+            const parsedUrl = new URL(resLink, xmut);
+            link = parsedUrl.href;
+        } else {
+            link = resLink;
+        }
+        const title = $('a', res).attr('title');
+        const pubDate = parseDate(resDate.text().trim());
+        return {
+            title,
+            link,
+            pubDate
+        };
+    }).get();
+    const itemPromises = itemsArray.map(async (item) => {
+        if (item.description) {
+            return item;
+        }
+        const res = await got(item.link, {
+            headers: {
+                referer: xmut,
+            },
+            https: {
+                rejectUnauthorized: false,
+            },
+        });
+        const $item = cheerio.load(res.data);
+        const content = $item('table #result #content form div #vsb_content_6').html();
+        item.description = content;
+        return item;
+    });
```

Comment:
You can take a look the docs. There's a specific section on [how to use cache](https://docs.rsshub.app/joinus/advanced/use-cache).

Reply:
Can I not use the cache? :3

Reply:
> You can take a look the docs. There's a specific section on [how to use cache](https://docs.rsshub.app/joinus/advanced/use-cache).

Can I not use the cache? :3

Reply:
Without adding cache, you are adding unnecessary presure to both RSSHub and target servers.
Take `/xmut/jwc/bkjw/jxyx` as an example, RSSHub will send 20 concurrent requests to the target servers if someone visit this route. If the returned results is cached, RSSHub won't send another 20 concurrent requests to the target servers when another person visited the same route.

Reply:
Roger, I'll commit my code later <3